### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tschanfeld**
per request in https://panw-global.slack.com/archives/C011J0AP7J8/p1704003947393949?thread_ts=1703729523.956839&cid=C011J0AP7J8 to add instructions on how to get the URL

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@tschanfeld Let's be aligned through all: README.md the parameter description in the integration yml file and the description.md.

**Comment by tschanfeld** [comment]
@ShahafBenYakir added proposed edits for YAML and description file.

File: Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI_description.md [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
-### URL
-1. In your Cortex XSIAM platform, go to **Settings** -> API Keys.
-2. Click the **Copy URL** button in the top right corner.
 </~XSIAM>
 
-<~XSOAR>
-
-In version 8.0.0 and above:
+<~XSOAR_SAAS>
 1. Go to Settings -> API Keys.
```
Comment (ShahafBenYakir) [inline Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI_description.md:11]: **Settings & Info** > **Settings** > **API Keys**.

Comment (ShahafBenYakir) [reply Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI_description.md:11]: Should be the same as https://github.com/demisto/content/pull/31930/files#diff-19e1bedf2f2bb5d3de2b4f11e5f1fcbc8b532f8712c4f9add1ad0c4faef6b787R32

File: Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
-    | Core Server URL |  | True |
+    | Core Server URL | For Cortex XSOAR 6.x, use the server URL. For Cortex XSOAR 8 and Cortex XSIAM, click the **COPY API URL** button on the API Keys page in **Integrations** > **API Keys** | True |
```
Comment (ShahafBenYakir) [inline Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/README.md:11]: Need to update the yml to have the same description https://github.com/demisto/content/pull/31930/files#diff-b2bf91e750c47a327f4c4cf45f76c35fe9fe6a661f0d39c0c20fde8cde98e513R14

**Review by ShahafBenYakir — COMMENTED** [review]
great

**Pull request merged**